pallets/stableswap/src/lib.rs

```diff
@@ -416,9 +423,18 @@ pub mod pallet {
 			Pools::<T>::try_mutate(pool_id, |maybe_pool| -> DispatchResult {
 				let mut pool = maybe_pool.as_mut().ok_or(Error::<T>::PoolNotFound)?;
 
+				ensure!(
+					pool.final_block <= current_block,
+					Error::<T>::AmplificationChangeNotCompleted
```

Comment:
we actually may want do do this, to cancel or start reverting some application as change could take months and situation may change to adjust it

Reply:
i was considering this. but as it is - the initial_amp would just to final. 

but now that i think of it - i can actually compute the correct amplification and set as initial.

Reply:
why would it jump ? you can configure it so it doesn't